app/views/users/show.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Here is a list of all posts from user 1 and bio</h1>
\ No newline at end of file
+<article class="main-article">
+  <div class="user-info-wrapper">
+    <%= image_tag @user.photo, :class => "user-avatar" %>
+    <div class="user-details">
+      <p><%= @user.name %></p>
+      <span>Number of posts: <%= @user.post_counter %></span>
+    </div>
+  </div>
+  
+  <div class="bio">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <div class="post">
+      <h4>Post #<%= index + 1 %></h4>
```

Comment:
- [ ] [Optional] Well! good job so far. As I mentioned previously, Here, you are showing a placeholder where the index is dynamically changing from `Post #1`, `Post #2`, and so on. However, in the wireframe, it shows the project title as `Post#1`, `Post#2`. I know the wireframe indicates the same as you have, but it's a suggestion to consider changing the placeholder to match the project title for a more user-friendly display.